### DISCUSSION TITLE
Is this community active ?

### DISCUSSION THREAD

**Discussion by lepinekong** [community/community#22945] [New to GitHub]
<p>There are very few views and rare answers. </p>

**Comment by lepinekong** [comment, marked as answer]
<p>OK never mind I understand it’s not stackoverflow :slight_smile:</p>
<p>Github support is great enough by email.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/lepinekong">@lepinekong</a>,</p>
<p>As you’ve noticed, this community is very new still. We’re looking to grow it and help it become more active, but these things do take time. I’m glad you were able to get the help you needed by contacting our Support team though. :slight_smile:</p>
<p>Thanks!</p>
